### PR TITLE
Banana Soap

### PR DESCRIPTION
**Pull request by Doggo5852**
## About the PR

Added banana soap, recipe 20u banana juice, 10u salt and 15u fat
## Why / Balance

paralyzeTime: 3.5, launchForwardsMultiplier: 2
I think it adds some makeshift way to make something better than soap. It has a somewhat difficult recipe. Has .5 more stun time than a banana peel but makes the player slip for more distance.
## Technical details

N/A
## Media

image

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

N/A
**Changelog**

:cl:
- add: Added banana soap

### CODE REVIEW

**Comment by Aidenkrz** [comment]
This doesnt have a construction graph, how do you craft it?

**Comment by Doggo5852** [comment]
Its in the "about the pr"

File: Resources/Textures/Objects/Specific/Janitorial/soap.rsi/meta.json
Comment (Piras314) [inline Resources/Textures/Objects/Specific/Janitorial/soap.rsi/meta.json:1]: banana soap sprite was created by you, right @Doggo5852 ? the copyright notice should then be: ```Banana sprites made by Doggo5852, rest taken from tgstation at https://github.com/tgstation/tgstation/commit/f8f4aeda930fcd0805ca4cc76d9bc9412a5b3428```

Comment (Doggo5852) [reply Resources/Textures/Objects/Specific/Janitorial/soap.rsi/meta.json:1]: the sprite was made by me, its basically the tgstation soap but yellow (and removed the text)

**Pull request merged**